x-pack/legacy/plugins/uptime/public/components/functional/location_map/embeddables/map_config.ts

```diff
@@ -62,7 +66,7 @@ export const getLowPolyLayer = () => {
   };
 };
 
-export const getDownPointsLayer = (downPoints: LocationPoint[]) => {
+export const getDownPointsLayer = (downPoints: LocationPoint[], dangerColor: string) => {
```

Comment:
I see we're modifying the map in this PR. Is that intentional? I thought this was only touching the left bar?

Reply:
Reading further I see this has some other fixes. That's fine, but it'd be good to update the issue description with a list of things that have changed in the UI with this PR.

Reply:
you are right, improving code is usually fine if you are touching some odd file unrelated to PR, but functionality changes deserve comment in PR description. I will add those.